src/libwrap/meson.build

```diff
@@ -1,16 +1,20 @@
 libwrap_sources =  files(
+	'adler32.h',
 	'cdma_parser.c',
 	'cdma_parser.h',
 	'cgdrv.c',
 	'disasm.c',
 	'disasm.h',
 	'drm.h',
+	'drm_fourcc.h',
```

Comment:
Doesn't look like this is the commit that adds drm_fourcc.h, is this in the wrong commit perhaps?

Reply:
Yep